lab_1/main.py

```diff
@@ -125,4 +162,14 @@ def sort_concordance(tokens: list, word: str, left_context_size: int, right_cont
     left_sort = True
     --> [['dog', 'is', 'happy', 'but', 'the', 'cat'], ['man', 'is', 'happy', 'the', 'dog', 'is']]
     """
-    pass
```

Comment:
There should be checks for input arguments. Smth like:
`if not isinstance(input_arg_name, expected_type):`
`do something here`